src/game/boe.main.cpp

```diff
@@ -227,19 +238,21 @@ int main(int argc, char* argv[]) {
 		menu_activate();
 		restore_cursor();
 
+		throw std::string { "fatal" };
```

Comment:
Obviously, this line will have to be removed before I can merge.

Reply:
Ahahahaha I can't believe I staged that